framework/src/test/java/org/checkerframework/framework/testchecker/sideeffectsonly/qual/Refined.java

```diff
@@ -0,0 +1,14 @@
+package org.checkerframework.framework.testchecker.sideeffectsonly.qual;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import org.checkerframework.framework.qual.SubtypeOf;
+
+@Documented
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ElementType.TYPE_USE, ElementType.TYPE_PARAMETER})
+@SubtypeOf({Unrefined.class})
+public @interface Refined {}
```

Comment:
This class (and `Unrefined`) have no documentation.  I'm not sure what they are.